.gitignore

```diff
@@ -164,3 +164,4 @@ _deps
 # Ignore all local history of files
 .history
 .ionide
+LowLevelGame
```

Comment:
huh?

Reply:
It’s the binary executable you get when building the project.

Reply:
The place the executable gets placed depends on how you build it, this should be added to your personal `.git/info/exclude` rather than the `.gitignore`

Reply:
Either way I don't think it hurts to add

Reply:
@Fripe070 ah I see. I don't know how cmake works :3